### ISSUE TITLE
Provide a non-markdown editor

### ISSUE THREAD

**Issue by tribut** [stefan-niedermann/nextcloud-notes#551]
The current markdown editor is almost unusable when editing lists. It randomly eats lines when hitting backspace at the start of a line. Copying from the text of a list item copies the indentation and bullet, even when the pasted line does not start with a `-` or `*`, and many more quirks. I constantly switch out of a note and back in just to see what I actually did and often I have to go to the web interface to restore an older version.
I know this is probably RxMarkdown's fault and there may be [better(?) options](https://github.com/stefan-niedermann/nextcloud-notes/issues/549) but as a stopgap (and maybe an alternative in general) would it be possible to provide an option to just use a normal text widget for editing?

**Comment by kaysond** [comment]
Bump. This is pretty awful

**Comment by stefan-niedermann** [comment]
I totally agree that RxMarkdown is not a good solution. But supporting Markdown is a basic feature of Nextcloud Notes and instead of removing it we should look for a better solution instead.

The last state of #549 is to analyse the "Markor"-Editor and have a look at what we can port to this app. It seems to be much more work than just including the RxMarkdown library, but it looks very promising.

Currently i am very busy with implementing Single Sign On and multi account handling. This comes with huge changes, so currently a Pull Request does not make sense (it only would produce large merge conflicts).

So, if you are interested in helping analyszing the Markor editor or creating a library on its code base, you are welcome to help over in #549, but

**TL;DR: It's not a goal to disable the Markdown editor, sorry**

**Comment by kaysond** [comment]
Is there a workaround for the issue with lists?

**Comment by stefan-niedermann** [comment]
I am not aware of any, but to be honest i never had this concrete issue:

> It randomly eats lines when hitting backspace at the start of a line

And for

> I constantly switch out of a note and back in just to see what I actually did

it should be enough to quickly switch to the view mode and back (the eye-icon in the toolbar).

**Comment by kaysond** [comment]
@stefan-niedermann If I create a new note with a markdown list such as:
```
- list item 1
- list item 2
```
Then tap the eye/view icon, it shows me the list with proper indentation an bullets. However, if I then attempt to edit the note again, it adds the bullets to the editor and I end up with something like:
```
· - list item 1
· - list item 2
```
These bullets and the indentation aren't actually editable, and they disappear if you delete the space after the dash. But it makes it very hard to read
